lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/app-doc-check.yaml

```diff
@@ -0,0 +1,69 @@
+name: DCMAW and Driving Licence
+description: >-
+  The combined journey for DCMAW and Driving Licence CRIs.
+entryEvents:
+  next:
+    targetState: CRI_DCMAW
+nestedJourneyStates:
+  CRI_DCMAW:
+    response:
+      type: cri
+      criId: dcmaw
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE
+      dl-auth-source-check:
+        targetState: CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK
+      not-found:
+        exitEventToEmit: incomplete-not-found
```

Comment:
This smells suspicious to me - why do we treat it differently in a mitigation journey 🤔 

I've asked in slack...

Reply:
note from slack: this should be treated the same as the other `incomplete` failures in the mitigation journey ie as a soft failure